packages/website/components/layout.js

```diff
@@ -8,12 +8,19 @@ import { useQuery } from 'react-query'
 import { useUser } from '../lib/user'
 
 const MaintenanceBanner = () => {
-  let maintenanceMessage = ''
+  let bannerMessage = ''
 
   const { data: statusPageData, error: statusPageError } = useQuery(
     'get-statuspage-summary',
     () => getStatusPageSummary()
   )
+
+  const incidents =
+    statusPageData?.incidents.filter(
+      (/** @type {{ resolved_at: string; }} */ incident) =>
+        incident.resolved_at === null
```

Comment:
Unless we are certain it will be null it can't hurt to check for falsey values including empty string here.